src/meta/src/barrier/progress.rs

```diff
@@ -281,10 +281,17 @@ impl CreateMviewProgressTracker {
     ) -> Option<TrackingCommand> {
         let actor = progress.chain_actor_id;
         let Some(epoch) = self.actor_map.get(&actor).copied() else {
-            panic!(
-                "no tracked progress for actor {}, is it already finished?",
+            // On restart, backfill will ALWAYS notify CreateMviewProgressTracker,
+            // even if backfill is finished on recovery.
+            // This is because we don't know if only this actor is finished,
+            // OR the entire stream job is finished.
+            // For the first case, we must notify meta.
+            // For the second case, we can still notify meta, but ignore it here.
+            tracing::info!(
+                "no tracked progress for actor {}, the stream job could already be finished",
                 actor
             );
+            return None;
```

Comment:
In this PR, is there any way to enter this branch? It seems we won't enter this branch until we support persisting 
streaming job meta before backfill is finished.

Reply:
Yes we won't enter this branch within this PR.
I will move this check to https://github.com/risingwavelabs/risingwave/pull/12167.

Reply:
Oh actually we will enter this branch. Because on restart, even finished partitions have to report their progress.
Otherwise the mview tracker won't be able to know these partitions are done, and will be stuck waiting for them.